ios/RNSScreenStack.mm

```diff
@@ -382,9 +382,13 @@ - (void)setModalViewControllers:(NSArray<UIViewController *> *)controllers
   [newControllers removeObjectsInArray:_presentedModals];
 
   // We need to find bottom-most view controller that should stay on the stack
-  // for the duration of transition. There are couple of scenarios:
+  // for the duration of transition.
+  // We will want to dismiss all
+
+  // There are couple of scenarios:
   // (1) No modals are presented or all modals were presented by this RNSNavigationController,
   // (2) There are modals presented by other RNSNavigationControllers (nested/outer)
```

Comment:
Just a small nit: add `,` at the end of this line 😄 :sorrry:

Reply:
https://github.com/software-mansion/react-native-screens/pull/2113/commits/0e1b5e606ab81042ca2d4f90af95779132a2a5dd 😅